### PR TITLE
pio-access: add page

### PR DESCRIPTION
**Pull request by marchersimon**
Part of #5517

### CODE REVIEW

File: pages/common/pio-access.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# pio access
+
+> Set the access level on published resources (packages) in the registry.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/access/index.html>.
```
Comment (bl-ue) [inline pages/common/pio-access.md:4]:
```suggestion
> More information: <https://docs.platformio.org/en/latest/core/userguide/access/>.
```

File: pages/common/pio-access.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Grant access to a resource for a specific user:
```
Comment (bl-ue) [inline pages/common/pio-access.md:6]:
```suggestion
- Grant a user access to a resource:
```

File: pages/common/pio-access.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`pio access grant {{guest|maintainer|admin}} {{username}} {{resource_urn}}`
+
+- Remove access to a resource for a specific user:
```
Comment (bl-ue) [inline pages/common/pio-access.md:10]:
```suggestion
- Remove a user's access to a resource:
```

File: pages/common/pio-access.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`pio access revoke {{username}} {{resource_urn}}`
+
+- Show all resources a user or team can access, along with the access level:
```
Comment (bl-ue) [inline pages/common/pio-access.md:14]:
```suggestion
- Show all resources that a user or team has access to and the access level:
```

File: pages/common/pio-access.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`pio access list {{username}}`
+
+- Set a resource to be privately accessible:
```
Comment (bl-ue) [inline pages/common/pio-access.md:18]:
```suggestion
- Restrict access to a resource to specific users or team members:
```

File: pages/common/pio-access.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`pio access private {{resource_urn}}`
+
+- Set a resource to be publicly accessible:
```
Comment (bl-ue) [inline pages/common/pio-access.md:22]:
```suggestion
- Allow all users access to a resource:
```

**Review by bl-ue — APPROVED** [review]
Some wording changes, not required though.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**